app/controllers/katello/api/v2/content_views_controller.rb

```diff
@@ -20,6 +20,31 @@ class Api::V2::ContentViewsController < Api::V2::ApiController
       param :component_ids, Array, :desc => N_("List of component content view version ids for composite views")
     end
 
+    api :POST, "/content_views/:id/export", N_("Export a content view to disk")
+    param :id, :identifier, :desc => N_("Content view identifier"), :required => true
```

Comment:
I"m not sure that incrementals for a CVV (that isn't the default organization view) make sense unless you are also removing things that got removed.  Library is always additive (in general), but different versions of a content view aren't.